### PR TITLE
Use promises

### PR DESCRIPTION
**Pull request by szantogeri**

### CODE REVIEW

File: examples/hotel-list.js [hunk 1]
```diff
@@ -2,25 +2,21 @@ var expedia = require("../lib/expedia")();
 
 // a complete list of options is available at http://developer.ean.com/docs/hotel-list/
 var options = {
-  "customerSessionId" : "thisisauniqueID",
   "customerIpAddress" : "127.0.0.1",
-  "customerUserAgent" : "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_4) AppleWebKit/537.36 (KHTML, like Gecko)",
+  "customerUserAgent" : "Node.js",
   "HotelListRequest": {
```
Comment (oldmonk7) [inline examples/hotel-list.js:7]: This contains the user agent, not sure if we can use Node.js. We can use the version `TravelBank/{api_version}`
 http://dev.ean.com/docs/common. I will implement the dynamic part.

Comment (szantogeri) [reply examples/hotel-list.js:7]: @oldmonk7 how can we get the api_version?

Comment (szantogeri) [reply examples/hotel-list.js:7]: User agent is just a string, it works with 'Node.js' as well.

Comment (oldmonk7) [reply examples/hotel-list.js:7]: We will figure out, where can we store the api-version, can be as simple as env var.
Meanwhile, you can keep it as `Node.js`. Trivial enough.

**Review by oldmonk7 — APPROVED** [review]

**Pull request merged**